### ISSUE TITLE
investigate our netlink integration with virtual interfaces provided by Docker

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2265]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2261
- Created at 2014-02-27 13:55:13 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed at 2020-03-24 14:10:53 as wontfix
- Assigned to nobody
---

Docker containers have virtual NICs per-container that we might get notifications from. We need to check if our netlink integration plays well with this use-case. We might choose to ignore all of them (since the networking is provided by the host) or only change online status of the domain that relates to the container in question. Needs a bit more research.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-02-27 15:15:59](https://pagure.io/SSSD/sssd/issue/2261#230826)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.12 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-02-27 15:16:23](https://pagure.io/SSSD/sssd/issue/2261#230827)**

Fields changed

rhbz:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-05-30 16:12:16](https://pagure.io/SSSD/sssd/issue/2261#230828)**

Moving to 1.12.1 for now, we can shuffle when we know more specifics about the Docker integration.

milestone: SSSD 1.12 beta => SSSD 1.12.1

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-07-02 18:57:52](https://pagure.io/SSSD/sssd/issue/2261#230829)**

Fields changed

review: 0 => 1

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-09-08 20:09:14](https://pagure.io/SSSD/sssd/issue/2261#230830)**

Mass-moving all tickets that didn't make 1.12.1 into 1.12.2

milestone: SSSD 1.12.1 => SSSD 1.12.2

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-09-30 19:06:30](https://pagure.io/SSSD/sssd/issue/2261#230831)**

We need to do a release as requested by downstream. Moving tickets that are not fixed already or very close to acking to 1.12.3

milestone: SSSD 1.12.2 => SSSD 1.12.3

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2014-10-30 12:17:19](https://pagure.io/SSSD/sssd/issue/2261#230832)**

This task is related to better container integration which is part of 1.13, moving

mark:  => 0
milestone: SSSD 1.12.3 => SSSD 1.13 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-10 16:47:18](https://pagure.io/SSSD/sssd/issue/2261#230833)**

Not required until we restart the work on the containerization.

milestone: SSSD 1.13 beta => SSSD 1.13 backlog

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-02-12 20:13:27](https://pagure.io/SSSD/sssd/issue/2261#230834)**

Mass-moving tickets not planned for the 1.13 release to 1.14

milestone: SSSD 1.13 backlog => SSSD 1.14 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-02-16 14:06:44](https://pagure.io/SSSD/sssd/issue/2261#230835)**

Stretch goal for 1.14

milestone: SSSD 1.14 beta => SSSD 1.14 backlog
sensitive:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-08-04 10:13:58](https://pagure.io/SSSD/sssd/issue/2261#230836)**

Please also see #2860 - just implementing the new option might be enough.

keywords:  => easyfix

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-12-15 11:00:54](https://pagure.io/SSSD/sssd/issue/2261#230837)**

Since the 1.14 branch is transitioning into maintenance mode and new functionality is being developed in master which will become 1.15 eventually, I'm mass-moving tickets from the 1.14 backlog milestone to the "Future releases" milestone.

milestone: SSSD 1.14 backlog => SSSD Future releases (no date set yet)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:42:40](https://pagure.io/SSSD/sssd/issue/2261#230838)**

**Metadata Update from @jhrozek**:
- Issue marked as blocked by: #2151
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-07-12 13:38:52](https://pagure.io/SSSD/sssd/issue/2261#448308)**

**Metadata Update from @jhrozek**:
- Custom field blocking reset (from 2151)
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review adjusted to on (was: 1)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: easyfix

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-13 13:22:28](https://pagure.io/SSSD/sssd/issue/2261#633134)**

**Metadata Update from @thalman**:
- Custom field design_review reset (from false)
- Custom field mark reset (from false)
- Custom field patch reset (from false)
- Custom field sensitive reset (from false)
- Custom field testsupdated reset (from false)
- Issue tagged with: Canditate to close

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:10:52](https://pagure.io/SSSD/sssd/issue/2261#636606)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2020-03-24 14:10:54](https://pagure.io/SSSD/sssd/issue/2261#636607)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
